icu4c/readme.html

```diff
@@ -33,7 +33,7 @@ <h1>
       <span class="only-rc">Release Candidate</span>
```

Comment:
Still left the RC stuff in here in case we ever want to have different READMEs in the future... otherwise we could remove it.

Reply:
Thanks!
This is probably fine for now. IIRC, I think there was some (brief) mention that maybe the README might be converted to Markdown and checked-in. (Though, I think it would mostly just point to the user-guide and download page).

---
In reply to: [398193398](https://github.com/unicode-org/icu/pull/1064#discussion_r398193398) [](ancestors = 398193398)